flipperdevices/flipperzero-firmware
# [FL-2911] IR Universal Audio Remote

# What's new

- Added a universal remote for audio players.
- Refactored Infrared CLI code to facilitate adding more universal remotes.

# Verification 

1. Compile and upload.
2. Go to `Infrared -> Universal Remotes -> Audio Players`.
3. Press all the buttons! The signals should be transmitted.
4. Exit the app and go to Flipper CLI.
5. Run `ir universal list audio`. A list of available signals should be printed.
6. Run `ir universal audio <signal_name>`, where `signal_name` is one of listed in p.5. The signals should be transmitted.

# Checklist (For Reviewer)
